pkg/sql/parser/testdata/show_function

```diff
@@ -0,0 +1,7 @@
+parse
+SHOW FUNCTION foo
+----
+SHOW FUNCTION foo
+SHOW FUNCTION (foo) -- fully parenthesized
+SHOW FUNCTION foo -- literals removed
+SHOW FUNCTION _ -- identifiers removed
```

Comment:
I'm not familiar with the parser but curious: Is it possible to write a `SHOW FUNCTION xxx` such that the fully parentesized, literals removed, and identifiers removed output are more non-trivial?

I guess I'm asking what's the most complicated yet legitimate `SHOW FUNCTION xxx`? E.g. can we do `SHOW FUNCTION defaultdb.foo` (db/sc that this function lives is provided), `SHOW FUNCTION defaultdb.sc1.foo(int)` (function argument is provided), `SHOW FUNCTION sc1.foo returns bool` (function return values provided; maybe I want the function named foo that returns bool)

Reply:
Ahh interesting. Maybe we want to name this `SHOW CREATE FUNCTION` then.

Reply:
I updated the syntax to `SHOW CREATE FUNCTION`. Let me know what you think.